pkg/kube/cluster-init.sh

```diff
@@ -502,12 +502,10 @@ if [ ! -f /var/lib/all_components_initialized ]; then
         fi
 
         if [ ! -f /var/lib/multus_initialized ]; then
-          if are_all_pods_ready; then
-            logmsg "Installing multus cni"
-            apply_multus_cni
-            # launch CNI dhcp service
-            /opt/cni/bin/dhcp daemon &
-          fi
+          logmsg "Installing multus cni"
```

Comment:
do we still need the 'are_all_pods_ready' check?
if apply_multus_cni actually fails, then we don't want to run the dhcp daemon, otherwise we'll end up with multiple copies of dhcp daemons.

Reply:
@naiming-zededa 

yeah I think we still want 'are_all_pods_ready' for cdi/longhorn install gating.  

I've pushed a fix to only start one dhcp daemon.